tests/installation/partitioning_raid.pm

```diff
@@ -263,10 +284,16 @@ sub run() {
     if (check_screen 'subvolumes-shadowed', 5) {
         send_key 'alt-y';
     }
-
-    if (get_var("LVM")) {
+    # check overview page for Suggested partitioning
+    if (get_var("LVM") and !get_var("UEFI")) {
```

Comment:
I tried to format your version and for me it's harder to read. I would like to stay on my version...
>     if (get_var("LVM") {
>         if (get_var("UEFI") {
>             assert_screen 'acceptedpartitioningraidlvmefi';
>         }
>         else {
>             assert_screen 'acceptedpartitioningraidlvm';
>         }
>     }
>     else {
>         if(get_var("UEFI") {
>             assert_screen 'acceptedpartitioningraidefi';
>         }
>         else {
>             assert_screen 'acceptedpartitioning';
>         }
>     }